### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by coderbyheart**
This PR tries to make the README more compact and readable.

- outdated vagrant information is removed
- the installation instruction is moved to a separate INSTALL.md
- the note about stylesheets is moved to the less folder
- info on how to contribute is moved to CONTRIBUTING.md (see https://github.com/blog/1184-contributing-guidelines)

Some smaller formatting fixes are added as bonus.

Preview: https://github.com/coderbyheart/Agora/blob/update-readme/README.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# How to contribute
+
+What to do? There is a companion project where the feature backlog lives. Go to the issues in [https://github.com/softwerkskammer/Agora-Backlog/issues](https://github.com/softwerkskammer/Agora-Backlog/issues)
+
```
Comment (NicoleRauch) [inline CONTRIBUTING.md]: This is by the way also very very outdated. I think we should remove this information for now. (and maybe groom or delete the backlog project...)

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# System requirements and Installation
+
+## Node.js
+
+It is a node.js project. Therefore you need node.js installed. Get it from [http://nodejs.org](http://nodejs.org).
+
+You need a current 4.x version to run the software (0.12.x should also be OK).
```
Comment (NicoleRauch) [inline INSTALL.md]: Currently we use Node.js 6 on the prod server. I'm not sure whether everything (i.e. dependencies) would still work with 0.12, so I'd rather shift this information (4 -> 6, 0.12 -> 4)

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+### Python
+
+* Python 2.7 (not Python 3!) in system path
+* For Mac OS: Python 2.7 is already installed.
+* For Windows:
+  * Make sure that Python and node.js are both either 32 bit or 64 bit - mixed combinations will not work!
+  * Add an environment variable `PYTHON` that points to the Python executable.
+
+### C++
```
Comment (NicoleRauch) [inline INSTALL.md]: Is this still required btw, now that we've removed the iconv project that depended on it? Are there other dependencies that still use C++?

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+* Copy `config/example-winston-config.json` to `config/winston-config.json`, remove the comment in the first line, and adapt the paths if you like.
+
+### Running the server
+
+* Open a shell in *REPO*
+* If your installation is fresh, perform the db initialization script:
+   * Run `node softwerkskammer/initialDBSetup`
+* If your installation is fresh or you changed anything:
+   * Run `npm test` (this creates the CSS files and other static contents and performs jshint checking)
```
Comment (NicoleRauch) [inline INSTALL.md]: Uh... we switched to eslint ages ago :-)

File: README.md [hunk 5]
```diff
@@ -1,233 +1,26 @@
...
-
-#### Grunt-CLI
+Want to help out? Check out our [Contribution guidelines](/CONTRIBUTING.md).
 
-We propose to install grunt-cli globally via the -g option of npm. To find out more about this option, see [https://npmjs.org/doc/global.html](https://npmjs.org/doc/global.html).:
+Tests
+-----
 
-* (Optional) To define the installation location of global npm packages on Unix-like Systems create a file called `.npmrc` with the following contents in your Home directory:
+After can run the tests with `npm test` (after you have [set up your local installation](/INSTALL.md)).
```
Comment (NicoleRauch) [inline README.md]: *You* can run...?

File: INSTALL.md (hunk 3 above)
Comment (leider) [inline INSTALL.md]: yes. can't remember which exactly, but emptying your node_modules and npm install from scratch would show you in the transcript

**Comment by leider** [comment]
why did you check in 3rd party logo? - I'd rather like to link it to jetbrains.

**Comment by coderbyheart** [comment]
I didn't find it in a linkable version on JetBrains …

**Comment by leider** [comment]
I see. Same here just now

File: INSTALL.md (hunk 3 above)
Comment (NicoleRauch) [inline INSTALL.md]: heapdump :-(

Shouldn't we remove that anyway?

File: INSTALL.md (hunk 4 above)
Comment (coderbyheart) [inline INSTALL.md]: Fixed in 3d7a81c28d2f91c78291266fb10b15588514cc9b

Btw, I suggest you look into https://www.npmjs.com/package/standard

File: README.md (hunk 5 above)
Comment (coderbyheart) [inline README.md]: Fixed in e847cad3d8ccb30849ecd746114f90915e9e96fb

File: CONTRIBUTING.md (hunk 1 above)
Comment (coderbyheart) [inline CONTRIBUTING.md]: Rephrased in 324f141c903520e2a8d53b0430d2044d8b9c10a7 (removed the link)

**Comment by NicoleRauch** [comment]
Looks good to me now. What do you think @leider ?

File: INSTALL.md (hunk 2 above)
Comment (coderbyheart) [inline INSTALL.md]: Added 4.x in 585dc0ff58f46dc04b2b00d54182a7cd4505123a

File: README.md [hunk 6]
```diff
@@ -1,233 +1,26 @@
...
-
-#### Grunt-CLI
+Want to help out? Check out our [Contribution guidelines](/CONTRIBUTING.md).
 
-We propose to install grunt-cli globally via the -g option of npm. To find out more about this option, see [https://npmjs.org/doc/global.html](https://npmjs.org/doc/global.html).:
+Tests
+-----
 
-* (Optional) To define the installation location of global npm packages on Unix-like Systems create a file called `.npmrc` with the following contents in your Home directory:
+Run the tests with `npm test` (after you have [set up your local installation](/INSTALL.md)).
```
Comment (NicoleRauch) [inline README.md]: Oh, one more thing: Can we make the installation instructions a bit more prominent? (I personally would probably not bother to read a "Tests" section when looking for the installation docs, tbh)

**Comment by coderbyheart** [comment]
@NicoleRauch changed the install note in 72268f9343150e7f0cc7a09ebf75283cd3990fac

**Pull request merged**